onnxruntime/core/framework/utils.cc

```diff
@@ -774,7 +774,8 @@ common::Status ExecuteGraph(const SessionState& session_state,
                             ExecutionMode execution_mode, const RunOptions& run_options,
                             const logging::Logger& logger) {
 #ifdef USE_CLOUD
-  if (run_options.config_options.configurations.count("use_cloud") &&
+  auto iter = run_options.config_options.configurations.find("use_cloud");
+  if (run_options.config_options.configurations.end() != iter &&
       run_options.config_options.configurations.at("use_cloud") != "0") {
```

Comment:
Why do you need to use 'at' here when you already have an iterator?

Reply:
Now resuing iterator.